public/app/features/dashboard/components/GenAI/utils.ts

```diff
@@ -158,3 +154,18 @@ export function getFilteredPanelString(panel: Panel): string {
 
   return JSON.stringify(filteredPanel, null, 2);
 }
+
+export function getPanelStrings(dashboard: DashboardModel): string[] {
+  const panelStrings = dashboard.panels
+    .filter(
+      (panel) =>
+        (panel.title.length > 0 && panel.title !== 'Panel title') || (panel.description && panel.description.length > 0)
```

Comment:
We could potentially add a check here for a description so if description is "description" or "panel description" then we filter it out. However, descriptions are not auto-created like titles.